Signal/src/call/PeerConnectionClient.swift

```diff
@@ -334,181 +377,220 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     public func createOffer() -> Promise<HardenedRTCSessionDescription> {
         SwiftAssertIsOnMainThread(#function)
+        let handle = self.handle
+        let (promise, fulfill, reject) = Promise<HardenedRTCSessionDescription>.pending()
```

Comment:
Since I've "unwrapped" the previously nested blocks, it's a lot easier to use:

```
let (promise, fulfill, reject) = Promise<HardenedRTCSessionDescription>.pending()
```

than:

```
return Promise { fulfill, reject in
```

The former lets us avoid passing around `fulfill` and `reject` which have lengthy type signatures.